tracks/javascript/exercises/space-age/mentoring.md

```diff
@@ -62,7 +62,9 @@ If a student is on an older version, kindly suggest they update to the newer tes
 
 - If they use the unary `+` to convert to a number, show them `Number(val)` which explicitely converts `val` to a
   number.
-- If they round via `Math.round(age * 100) / 100`, introduce them to [`Number#toFixed`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed).
+  
+### Other considerations
+- While [`Number#toFixed`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed), is shorter to write, using `Math.round` is actually better performance-wise.
```

Comment:
```suggestion
- While [`Number#toFixed`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed), is shorter to write, using `Math.round` has better performance
```.

Reply:
The _length_ of `toFixed` isn't really important. `toFixed` is much more declarative than `Math.round(n * Math.pow(10, digits)/Math.pow(10, digits))`. So whilst `Math.round` is faster, the intention (rounding on _2_ digits) is much harder to follow.

In general, if people want to round using `Math.round`, I suggest writing a helper:

```javascript
function roundOnDigits(n, digits) {
  return Math.round(n * Math.pow(10, digits)/Math.pow(10, digits))
}
```
---

I have no problem adding the consideration (it should be a Talking point, not a new heading), but `length` is not the reason `toFixed` is recommended.